src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -713,11 +713,37 @@ If you want to disable Elasticsearch and use the SQL database as the default bac
 audit.elasticsearch.enabled=false
 ```
 
-## Rebuilding the Repository Index
+## Rebuilding the Repository Index{{> anchor 'reindex'}}
 
-If you need to reindex the whole repository, you can do this from the [Nuxeo Dev Tool Browser Extension]({{page page='nuxeo-dev-tools-extension'}}#features).
+If you need to reindex the whole repository, you have different possibilities:
 
-You can fine tune the indexing process using the following options:
+### Re-index the repository using the WorkManager (the legacy way)
+ 
+There are 3 ways to run it:
+ 
+1. you can do this from the [Nuxeo Dev Tool Browser Extension]({{page page='nuxeo-dev-tools-extension'}}#features).
+
+2. you can do that from the JSF UI > Admin center > Elasticsearch > Admin
+
+3. you can simply use `curl`
+
+```bash
+curl -X POST "<NUXEO_URL>/nuxeo/site/automation/Elasticsearch.Index" -u Administrator:<PASSWORD> -H 'content-type: application/json+nxrequest' -d '{"params":{},"context":{}}'
```

Comment:
`application/json+nxrequest` is deprecated, `application/json` should be used instead, and other occurrences below